recipes/cgal/all/conanfile.py

```diff
@@ -1,7 +1,9 @@
 import os
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
-
+from conan import ConanFile
+from conan.tools import files
+from conan.tools import scm
+from conan.errors import ConanInvalidConfiguration
+from conans import CMake
```

Comment:
```suggestion
from conans import CMake

required_conan_version = ">=1.50.0"


```
conan.errors requires 1.50

Reply:
Actually, the import of `ConanInvalidConfiguration` was reported by the linter as no longer used. With latest push (f66e74ae4617146857cb5f442b2c51df63a58e57) I have removed that line of import.

Reply:
`self.info.clear()` also requires 1.50: https://docs.conan.io/en/latest/changelog.html#jun-2022